politeiawww/cmd/politeiawwwcli/commands/cmsedituser.go

```diff
@@ -16,6 +16,7 @@ import (
 
 // CMSEditUserCmd updates a user's information.
```

Comment:
This comment needs to explain the different behaviors of this route.  It's not very obvious anymore.  Same for the API docs.